zio-cli/shared/src/main/scala/zio/cli/completion/PrimTypeCompletion.scala

```diff
@@ -0,0 +1,61 @@
+package zio.cli.completion
+
+import zio._
+import zio.blocking.Blocking
+import zio.cli.PathType
+import zio.cli.PrimType
+
+import java.time.{ZoneId => JZoneId}
+import scala.jdk.CollectionConverters._
+
+object PrimTypeCompletion {
+  def firstTokens(primType: PrimType[Any], prefix: String, compgen: Compgen): RIO[Blocking, Set[String]] =
```

Comment:
Here and elsewhere, ZIO no longer requires that `Blocking` be in the ZIO environment. Just use the ZIO.*blocking* methods to avoid putting the blocking thread pool in the environment.

Reply:
👍 Per our discussion on Wednesday, I will leave this for now and instead modify [zio-process](https://github.com/zio/zio-process/) to make it possible to eliminate `Blocking` here.